src/common/refiners/ForwardDateRefiner.ts

```diff
@@ -17,6 +17,27 @@ export default class ForwardDateRefiner implements Refiner {
         results.forEach(function (result) {
             let refMoment = dayjs(context.refDate);
 
+            if (result.start.isOnlyTime() && refMoment.isAfter(result.start.dayjs())) {
+                refMoment = refMoment.add(1, "day");
+
+                result.start.imply("day", refMoment.date());
```

Comment:
Nit: Could you add `implySimilarDate()` (see. `assignSimilarDate()`) in the `util/dayjs` and use the function here?